specs/system_config.md

```diff
@@ -111,3 +115,32 @@ The contained log events are filtered and processed as follows:
 Note that individual derivation stages may be processing different L1 blocks,
 and should thus maintain individual system configuration copies,
 and apply the event-based changes as the stage traverses to the next L1 block.
+
+## Ecotone Activation
+
+### Constants
+
+The initial fee paramaters for the Ecotone hardfork are noted in the following table.
+
+| `l1BasefeeScalar` | `l1BlobBasefeeScalar` |
+|---------------------------------------------------|---|
+| Value of `scalar` at Ecotone activation block - 1 | 0 |
```

Comment:
From the phrasing, it's a bit ambiguous if one is being subtracted from: 
1. `scalar`
2. the "Ecotone activation block"

Reply:
I updated this